p4-evaluate-news-nlp/src/client/js/formHandler.js

```diff
@@ -30,7 +43,10 @@ function handleSubmit() {
 
   if (urlCheck) {
     try {
-      getApiKey();
+      getApiKey().then((apiKey) => {
+        return getApiCall(apiKey.api);
+      });
+      //.then((data) => {});
```

Comment:
don't include debugging comments in commits

Reply:
done in [fa51653](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/8/commits/fa516536cb6f3f2079108daa9df3466a2e84a077)

Reply:
@leemoispace wrong link 😄